docker-compose.yml

```diff
@@ -2,27 +2,29 @@ version: "3"
 
 services:
 
-  bash:
-    image: quay.io/natlibfi/annif
-    volumes:
-      - ${ANNIF_PROJECTS}:/annif-projects
-    user: ${MY_UID}:${MY_GID}
-    stdin_open: true
-    tty: true
-    command: bash
-
-  gunicorn_server:
-    image: quay.io/natlibfi/annif
+  annif_app:
+    image: quay.io/natlibfi/annif:latest
```

Comment:
I see that this changed to `:latest`, was this intended?

Reply:
Yes, the tag is `latest` by default; this just makes the behaviour more explicit.